src/components/FileMenu/__tests__/GetLinkDialog.spec.js

```diff
@@ -84,28 +78,26 @@ describe('The FileMenu - GetLinkDialog component', () => {
                 baseUrl,
             })
 
-            const href = getGetLinkDialogComponent({
-                type,
-                id,
-                onClose,
-            })
-                .find('a')
-                .prop('href')
+            props.type = type
+            props.id = id
 
-            expect(href).toMatch(expected)
+            render(<GetLinkDialog {...props} />)
```

Comment:
And here we could just do:
```<GetLinkDialog id={id} type={type} />```

Reply:
Refactored to not use the shared props object.